benefits/core/models/transit.py

```diff
@@ -27,6 +27,67 @@ def agency_logo_large(instance, filename):
     return _agency_logo(instance, filename, "lg")
 
 
+class TransitProcessorConfig(models.Model):
```

Comment:
I'm sort of confused what this gets us, other than not having to redefine the single `environment` field on each model? 

I.e. the `Environment` class can be defined at the top-level and used by both `*Config.environment` fields. Is there some other advantage with the ABC?

Reply:
I dropped the commit and force-pushed [just now](https://github.com/cal-itp/benefits/pull/2854#event-17339735842), and then I moved `Environment` to the top-level (https://github.com/cal-itp/benefits/pull/2854/commits/7154433bf667aa8b016c4fb69b74c84db98cbdaa). Let me know what you think!